src/rgw/motr/gc/gc.cc

```diff
@@ -212,41 +337,52 @@ int MotrGC::get_locked_gc_index(uint32_t& rand_ind) {
     if (rc == 0)
       break;
   }
+  rc = 0; // remove this line after lock implementation
   rand_ind = new_index;
   return rc;
 }
 
-int MotrGC::list(std::vector<motr_gc_obj_info>& gc_entries) {
+int MotrGC::list(std::vector<std::unordered_map<std::string, std::string>> &gc_entries) {
   int rc = 0;
   int max_entries = 1000;
+  max_indices = get_max_indices();
```

Comment:
MotrGC has max_indices member, do we still needs this function ?

Reply:
The `list()` function was unable to pick up the updated value of `max_indices` as the object that initializes this attribute and object which calls list function are different. Hence created a function so that we get updated value whenever we need it.